faculty_cli/cli.py

```diff
@@ -558,16 +575,36 @@ def list_servers(project, all, verbose):
                 server.created_at.strftime("%Y-%m-%d %H:%M"),
             )
         )
-    if not found_servers and verbose:
+
+    if not found_servers:
         click.echo("No servers.")
-    elif project and verbose:
-        servers = [server[1:] for server in found_servers]
-        click.echo(tabulate(servers, headers[1:], tablefmt="plain"))
-    elif project or not verbose:
+        return
+
+    if format == "table":
+        if project:
```

Comment:
I know this was missing from the current implementation, but could you include a comment that this is removing the project column from the output? It's not clear from the implementation.